src/main.rs

```diff
@@ -1,7 +1,7 @@
 use ckb_bin::run_app;
 use ckb_build_info::Version;
 
-#[cfg(unix)]
+#[cfg(all(unix, not(target_os = "macos")))]
```

Comment:
```suggestion
#[cfg(all(not(target_env = "msvc"), not(target_os = "macos")))]
```

As suggested in https://github.com/gnzlbg/jemallocator